src/cmd/singularity/cli/actions.go

```diff
@@ -206,6 +207,20 @@ func execStarter(cobraCmd *cobra.Command, image string, args []string, name stri
 		engineConfig.SetImage(abspath)
 	}
 
+	if Nvidia {
+		NvidiaBindPaths, err := nvidiautils.GetNvidiaBindPath(buildcfg.SINGULARITY_CONFDIR)
+		if err != nil {
+			sylog.Infof("Unable to capture nvidia bind points: %v", err)
+		} else {
+			if len(BindPaths) == 0 {
+				sylog.Warningf("Could not find any NVIDIA libraries on this host!")
+				sylog.Warningf("You may need to edit %v/singularity/nvliblist.conf", buildcfg.SINGULARITY_CONFDIR)
```

Comment:
This should be:  `%v/nvliblist.conf` ...   buldcfg.SINGULARITY_CONFDIR includes the singularity/ directory